packages/InteractiveProfilingTool-Tests.package/IPTTreeModelTestCase.class/instance/testGetIconIfBelowThreshold.st

```diff
@@ -0,0 +1,12 @@
+tests
+testGetIconIfBelowThreshold
+
+	| model messageTally |
+	
+	model := IPTTreeModelMock new. 
+	messageTally := MessageTallyMock new. 
+
+	messageTally tally: 49.
+	model totalTime: 100.
+	
+	self assert: nil equals: (model getIcon: messageTally)
```

Comment:
In the other test methods you introduce variables `actual` and `expected`. I think the variables greatly improve the readability of the tests. Please consider using `actual` and `expected` in this test too.

Reply:
I decided not to introduce these variables here as the assertion itself is already quite short and ease to read in my opinion.